src/main/java/deanery/exception/WrongNumber.java

```diff
@@ -0,0 +1,7 @@
+package deanery.exception;
+
+public class WrongNumber extends Exception{
+    public WrongNumber(String errorMessage){

```

Comment:
if you wanted to make this exception parametrized, you could for example provide the number and a student as the parameters and then write in the message "Room A123 is not correct for student 23456"